src/components/editMeasure/populationCriteria/groups/QICore/QICoreMeasureGroups.tsx

```diff
@@ -263,6 +274,15 @@ const MeasureGroups = (props: MeasureGroupProps) => {
 
   useEffect(() => {
     if (measure?.groups && measure?.groups[measureGroupNumber]) {
+      // manually sort group populations in existing groups on retrieval
+      const group = measure?.groups[measureGroupNumber];
+      group.populations.sort((a, b) => {
+        return (
+          allPopulations.findIndex((all) => all.name === a.name) -
+          allPopulations.findIndex((all) => all.name === b.name)
```

Comment:
Do we have to do this for QDM MeasureGroups as well ?

Reply:
Surprisingly not. For whatever reason just sorting the array order helpers fixed it for QDM.